ts/features/payments/checkout/components/CheckoutPaymentMethodsList.tsx

```diff
@@ -51,6 +55,19 @@ const CheckoutPaymentMethodsList = () => {
     O.getOrElse(() => 0)
   );
 
+  const latestPaymentMethodListItem = useMemo(
+    () =>
+      pipe(
+        latestPaymentUsed,
+        O.map(mapUserWalletToRadioItem),
+        O.map(Array.of),
+        O.map(A.map(O.fromNullable)),
+        O.map(A.compact),
+        O.getOrElse(() => [] as Array<RadioItem<string>>)
```

Comment:
```suggestion
        latestPaymentUsed,
        O.chainNullableK(mapUserWalletToRadioItem),
        O.map(A.of),
        O.getOrElse(() => [] as Array<RadioItem<string>>)
```

This is clearer, what do you think?

Reply:
You're an fp-ts wizard! 🧙‍♂️

Addressed into 4e22fe4be0d5c23ab4d99c02c31cc5a9c8c42763